test/built-ins/Array/prototype/includes/values-are-not-cached.js

```diff
@@ -20,16 +20,8 @@ function getCleanObj() {
     get: function() {
       Object.defineProperty(obj, "0", {
         get: function() {
-          Object.defineProperty(obj, "1", {
-            get: function() {
-              return "ecma262";
-            }
-          });
-          Object.defineProperty(obj, "2", {
-            get: function() {
-              return "cake";
-            }
-          });
+          obj[1] = "ecma262";
+          obj[2] = "tc39";
```

Comment:
As nice as the simplification is, this change invalidated the test. The final assertion in this file expects `obj[2]` to have the value `"cake"`, but (after this change) it actually takes the value `"tc39"`.